Microsoft.Data.Tools.Diagnostics.Tracer Information: 0 : 2023-06-12T17:32:37 : Initialized ModelSchema - elements 148
Microsoft.Data.Tools.Diagnostics.Tracer Information: 19 : 2023-06-12T17:32:37 : IsolatedStorageFile probe success
Microsoft.Data.Tools.Diagnostics.Tracer Information: 0 : 2023-06-12T17:32:37 : RegistryManager.DatabaseLockTimeout: 5 seconds.
Microsoft.Data.Tools.Diagnostics.Tracer Information: 0 : 2023-06-12T17:32:37 : RegistryManager.DatabaseQueryTimeout: 60 seconds.
Microsoft.Data.Tools.Diagnostics.Tracer Information: 0 : 2023-06-12T17:32:37 : RegistryManager.DatabaseLongRunningQueryTimeout: 0 seconds.
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : Logging Ambient Settings...
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : DatabaseLockTimeout: 5000
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : QueryTimeout: 60
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : LongRunningQueryTimeout: 0
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : AlwaysRetryOnTransientFailure: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : ConnectionRetryHandler: 
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : MaxDataReaderDegreeOfParallelism: 8
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : TraceRowCountFailure: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : TableProgressUpdateInterval: 300
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : UseOfflineDataReader: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : StreamBackingStoreForOfflineDataReading: File
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : ModelFilePath: 
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : MasterReferenceFilePath: 
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : DisableIndexesForDataPhase: True
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : ReliableDdlEnabled: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : ImportModelDatabase: True
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : SupportAlwaysEncrypted: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : AlwaysEncryptedWizardMigration: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : AlwaysEncryptedWizardPreserveTable: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : AlwaysEncryptedWizardUseOnlineApproach: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : AlwaysEncryptedWizardDowntimeInSeconds: 300
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : AlwaysEncryptedWizardMaxDivergingIterations: 5
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : AlwaysEncryptedWizardMaxIterations: 100
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : AlwaysEncryptedWizardMaxIterationDurationInDays: 3
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : SkipObjectTypeBlocking: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : DoNotSerializeQueryStoreSettings: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : LargeObjectHeapCompaction: False
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : LargeObjectHeapCompactionIntervalInSeconds: 30
Microsoft.Data.Tools.Diagnostics.Tracer Warning: 0 : 2023-06-12T17:32:37 : ReferencePaths: System.String[]
Microsoft.Data.Tools.Diagnostics.Tracer Verbose: 1 : 2023-06-12T17:32:37 : Entering ExtensionTypeLoader.LoadTypes().
Microsoft.Data.Tools.Diagnostics.Tracer Error: 19 : 2023-06-12T17:32:53 : Retry requested: Retry count = 1. Delay = 00:00:00, SQL Error Code = -2146232060, SQL Error Number = 53, Can retry error = False, Will retry = False System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Microsoft.Data.Tools.Schema.Common.SqlClient.ReliableSqlConnection.<OpenConnection>b__0()
   at Microsoft.Data.Tools.Schema.Common.SqlClient.RetryPolicy.<>c__DisplayClass4.<ExecuteAction>b__3(RetryState retryState)
   at Microsoft.Data.Tools.Schema.Common.SqlClient.RetryPolicy.ExecuteAction[R](Func`2 func, Nullable`1 token)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
Microsoft.Data.Tools.Diagnostics.Tracer Information: 1 : 2023-06-12T17:32:53 : A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)
Microsoft.Data.Tools.Diagnostics.Tracer Error: 19 : 2023-06-12T17:33:07 : Microsoft.SqlServer.Dac.DacServicesException: Could not connect to database server. ---> System.Data.SqlClient.SqlException: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception: The network path was not found
   --- End of inner exception stack trace ---
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Microsoft.SqlServer.Dac.DacServices.GetDatabaseSchemaProvider(SqlConnectionFactory connectionFactory)
   --- End of inner exception stack trace ---
Microsoft.Data.Tools.Diagnostics.Tracer Error: 19 : 2023-06-12T17:33:07 : SqlException - Client Connection Id: 00000000-0000-0000-0000-000000000000, Error Number: 53, State: 0, Class: 20 System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---> System.ComponentModel.Win32Exception (0x80004005): The network path was not found
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreateNonPooledConnection(DbConnection owningConnection, DbConnectionPoolGroup poolGroup, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at Microsoft.SqlServer.Dac.DacServices.GetDatabaseSchemaProvider(SqlConnectionFactory connectionFactory)
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
Microsoft.Data.Tools.Diagnostics.Tracer Information: 0 : 2023-06-12T17:33:07 : SqlPackage completed
